src/main/scala/org/apache/mesos/chronos/scheduler/api/Iso8601JobResource.scala

```diff
@@ -46,6 +46,10 @@ class Iso8601JobResource @Inject()(
           "the job's name is invalid. Allowed names: '%s'".format(JobUtils.jobNamePattern.toString()))
         if (!Iso8601Expressions.canParse(newJob.schedule, newJob.scheduleTimeZone))
           return Response.status(Response.Status.BAD_REQUEST).build()
+        if(! JobUtils.isValidURIDefinition(newJob)) {
```

Comment:
In terms of style across the repo there is a space following each 'if' statement but in your commit every single 'if' lacks the space between the opening parentheses. Consider adding the space to keep consistent style and removing the space between the negation (!) and statement.